src/utils.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import path from 'path';
+import fs from 'fs';
+import { ReportPortalConfig } from './models';
+import { RP_CONFIG_FILE_NAME } from './constants';
+
+export const getConfig = (providedConfig?: ReportPortalConfig): ReportPortalConfig => {
```

Comment:
When this function can receive `providedConfig`? According to your reporter's implementation this parameter will always `undefined`.

Reply:
This provided config needs for tests. In them we create a new instance of the class.
In the case of a reporter, we can't import insansce of the class.
`Error: src\index.ts: reporter file must export a single class.`